scripts/lib/fix-standalone-certbot

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
+
+hostnames=$(grep -l 'authenticator = standalone' /etc/letsencrypt/renewal/*.conf | sed 's/.*\///; s/\.conf$//')
+
+for hostname in $hostnames; do
+    # Force a cert renewal to force the config file to update
+    /usr/bin/certbot certonly --webroot --webroot-path=/var/lib/zulip/certbot-webroot/ --force-renewal -d "$hostname"
```

Comment:
I guess we also need to update `setup-certbot` to use `--post-hook 'service nginx reload'`, and the Puppet test to check whether that’s missing.

Reply:
We install a post-deploy hook in`/etc/letsencrypt/renewal-hooks/deploy/001-nginx.sh` rather than via the configuration file (done in 01e8f752a8c2b4d86f3b6b5d130c27c5d8231cf4).

Reply:
And we need to run `service nginx reload` by hand because `certonly` doesn't trigger any post-renew hooks, since it's "cert only."